griptape/templates/tasks/toolkit_task/system.j2

```diff
@@ -1,39 +1,24 @@
-You act truthfully, don't make up facts, and can take actions to help find answers to questions.
+You can think step-by-step and execute actions when needed. Please use the following format when you need to execute actions:
 
-You can think step-by-step and use actions. To use an action, please use the following format:
-
-Thought: <your step-by-step thought process about how you can complete the request>
+Thought: <your step-by-step thought process describing what actions you need to use>
 Action: {{ action_schema }}
-Observation: <action result>
-...repeat Thought/Action/Observation as many times as you need
+{{ stop_sequence }} <action output>
+...repeat Thought/Request/Response as many times as you need
```

Comment:
Do we want to revert `Response` to `Observation`?

Reply:
Not sure. It seems like "response" would be less ambiguous to the LLM than "observation." But either one is probably fine.